### ISSUE TITLE
Button does not have required alt text

### ISSUE THREAD

**Issue by rmainseas** [seas-computing/mark-one#111]
### Description
Button components in `mark-one` do not currently require `alt={}` attributes. This means that it's possible to miss out (or forget) accessibility features like alt text. Going forward we should really add this as a required prop to make buttons more accessible by default.

---

- [ ] Test
- [ ] Docs

**Comment by sonusindhu** [comment]
If I am not wrong, you are referring to `aria-label` for `alt={}`

**Comment by sonusindhu** [comment]
@rmainseas how do you think, it should be implemented?
Just add optional property for now or we need to make it required?
